TL/28_utility_libraries_data_faker_awaitility/Utility_Libraries_README.adoc

```diff
@@ -1,21 +1,16 @@
 = Utility libraries
----
+:doctype: book
+:icons: font
+:toc: left
+:sectnums:
```

Comment:
please add for source codes next actions:
`:source-highlighter: highlightjs
:highlightjs-languages: <all languages that are used in the source code>`

Reply:
added, thank you!